### PR TITLE
pin six version to 1.10.0 for shade

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

File: roles/common/tasks/python.yml [hunk 1]
```diff
@@ -51,6 +51,19 @@
     register: result
     until: result|succeeded
     retries: 5
+
+  - name: install shade bits for ansible modules
+    pip:
+      name: "{{ item }}}
```
Comment (ddaskal) [inline roles/common/tasks/python.yml]: typo, this errored for me. change `"{{ item }}}` to `"{{ item }}"`

File: roles/common/tasks/python.yml [hunk 2]
```diff
@@ -51,7 +77,8 @@
     register: result
     until: result|succeeded
     retries: 5
-  when: openstack_install_method == 'distro'
+
+when: openstack_install_method == 'distro'
```
Comment (nirajdp76) [inline roles/common/tasks/python.yml]: seems like need indentation to line up with block:

File: roles/common/tasks/python.yml [hunk 3]
```diff
@@ -40,6 +57,15 @@
       extra_args: "{{ pip_extra_args }} --upgrade"
     register: result
     until: result|succeeded
+    #retries: 5
```
Comment (zfeldstein) [inline roles/common/tasks/python.yml]: This should be un-commented.

**Comment by zfeldstein** [comment]
LGTM 👍

**Comment by nirajdp76** [comment]
LGTM

**Pull request closed without merging**